css/styles.css

```diff
@@ -342,6 +499,20 @@ button {
   border: 0;
```

Comment:
- [ ] Kindly add appropriate background color to all button kindly check figma design 👍🏿